lumicks/pylake/channel.py

```diff
@@ -194,11 +203,17 @@ def from_dataset(dset, y_label="y"):
 
     @property
     def start(self):
-        return self.timestamps[0]
+        if len(self.timestamps) > 0:
```

Comment:
The Pythonic way to check for a non-empty sequence is a simple `if self.timestamps:`, because an empty sequence evaluates to `False`.

Reply:
The issue was that timestamps usually has more than one element, which raises an error that this is ambiguous. Would you prefer self.timestamps.any() over the length check? I thought the len here was more clear about the intent than self.timestamps.any() is.

Reply:
Ah, I forgot that these were `numpy` arrays which don't quite do the same thing with regard to `True`ness as regular sequences in Python. Never mind, this is fine.

Reply:
👍